src/main/java/teammates/storage/api/FeedbackSessionsDb.java

```diff
@@ -184,6 +184,7 @@ public void addInstructorRespondent(String email, FeedbackSessionAttributes feed
         addInstructorRespondents(emails, feedbackSession);
     }
 
+    @SuppressWarnings("PMD.AvoidThrowingRawExceptionTypes") //Transactions do not support throwing checked exceptions
```

Comment:
It is not transactions do not support. Should be `// The objectify library does not support throwing checked exceptions inside transactions`